pages/k8s/operations.md

```diff
@@ -0,0 +1,278 @@
+---
+wrapper_template: "base_docs.html"
+markdown_includes:
+  nav: "shared/_side-navigation.md"
+context:
+  title: "Basic operations"
+  description: How to operate your CDK cluster.
+keywords: dashboard, juju,kubectl,config
+tags: [operating]
+sidebar: k8smain-sidebar
+permalink: operations.html
+layout: [base, ubuntu-com]
+toc: False
+---
+
+Now you have installed your **CDK** cluster, what can you do with it? This page details
+some basic cluster operations, including how to check the status of your cluster and test
+that it is working with the built-in demo deployment.
+
+If you are already familiar with this, you may still like to check out the [Next steps][next]
+for more useful guides on operating CDK.
+
+## Install and configure kubectl
+
+You will need **kubectl** to be able to use your Kubernetes cluster. If it is not already
+installed (it is automatically  intalled if you used conjure-up to deploy **CDK**), it is easy
```

Comment:
installed

Reply:
This was obviously the one typo I habitually leave to make you feel you have accomplished something :smiley: